democracy/src/migrations.rs

```diff
@@ -0,0 +1,136 @@
+use super::*;
+
+use frame_support::{pallet_prelude::*, storage_alias, traits::OnRuntimeUpgrade};
+
+/// The log target.
+const TARGET: &str = "democracy::migration::v1";
+
+mod v0 {
+	use super::*;
+	use encointer_primitives::democracy::ProposalActionIdentifier;
+
+	#[storage_alias]
+	pub(super) type CancelledAt<T: Config> =
+		StorageMap<Pallet<T>, Blake2_128Concat, ProposalActionIdentifier, u64, OptionQuery>;
+}
+
+pub mod v1 {
+	use super::*;
+
+	#[allow(dead_code)]
+	pub struct MigrateV0toV1purging<T>(sp_std::marker::PhantomData<T>);
```

Comment:
They have now the `VersionedMigration<V1, V2>` which runs the pre/post version check internally in substrate, but it doesn't really matter, this is just an FYI

Reply:
thx